backend/lucene/src/main/java/org/hibernate/search/backend/lucene/logging/impl/Log.java

```diff
@@ -595,4 +595,14 @@ SearchException conflictingObjectFieldModel(String absoluteFieldPath,
 			value = "Multiple conflicting models for field '%1$s': '%2$s' vs. '%3$s'.")
 	SearchException conflictingFieldModel(String absoluteFieldPath,
 			LuceneIndexSchemaObjectNode objectNode, LuceneIndexSchemaFieldNode fieldNode, @Param EventContext context);
+
+	@Message(id = ID_OFFSET_2 + 107, value = "Query took longer than expected: '%1$s microseconds'. Query: '%2$s'.")
+	SearchException timedOut(Long microseconds, String queryDescription);
```

Comment:
I think there is no message id for this error message. Because it seems that was not generated using a `Log` class. The exception was generated in this way:

``` java
throw timeoutExceptionFactory.createTimeoutException(
  "Full-text query took longer than expected (in microsecond): " + 
  TimeUnit.NANOSECONDS.toMicros( elapsedTime ),String.valueOf( query )
);
```

and in the default factory we have:

``` java
@Override
public RuntimeException createTimeoutException(String message, String queryDescription) {
  return new QueryTimeoutException( message, queryDescription );
}
```

Isn't it?